dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/BrowserAPI.java

```diff
@@ -180,7 +187,7 @@ public WfData(Contentlet contentlet, List<Integer> permissions, User user, boole
 	public Map<String, Object> getFolderContent(User user, String folderId,
 			int offset, int maxResults, String filter, List<String> mimeTypes,
 			List<String> extensions, boolean showWorking, boolean showArchived, boolean noFolders,
-			boolean onlyFiles, String sortBy, boolean sortByDesc, boolean excludeLinks, long languageId)
+			boolean onlyFiles, String sortBy, boolean sortByDesc, boolean excludeLinks, long languageId, boolean dotAssets)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'onlyFiles' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5337824)